### PR TITLE
fix wrong counting of PyObject

### PR DESCRIPTION
**Pull request by PaulRousse**

### CODE REVIEW

**Review by p-e-s — APPROVED** [review]
Ok for me. I let the experts check it.

File: pyopendds/dev/include/pyopendds/basictype.hpp [hunk 1]
```diff
@@ -29,6 +29,7 @@ class BooleanType {
         } else {
             py = Py_True;
         }
+        Py_INCREF(py);
```
Comment (Sdpierret) [inline pyopendds/dev/include/pyopendds/basictype.hpp:32]: the REF is never released ?

**Review by Sdpierret — COMMENTED** [review]
only one question

**Review by Sdpierret — COMMENTED** [review]
only one question

File: pyopendds/dev/include/pyopendds/basictype.hpp (hunk 1 above)
Comment (Sdpierret) [reply pyopendds/dev/include/pyopendds/basictype.hpp:32]: https://docs.python.org/3/c-api/bool.html

maybe use `PyBool_FromLong(cpp)` and remove the `Py_INCREF()`

**Review by Sdpierret — CHANGES_REQUESTED** [review]

**Review by Sdpierret — APPROVED** [review]

**Pull request merged**